### DISCUSSION TITLE
Option to turn action bars off.

### DISCUSSION THREAD

**Discussion by Firegodd** [siweia/NDui#161] [Q&A] [answered]
Can we get an option to turn certain single action bars off? I like using Bartender as it has this feature, if we can get this with Ndui I will get rid of Bartender all together.

**Comment by siweia** [comment, marked as answer]
You can turn off actionbars in blizzard interface options panel. Press ESC-Interface, under actionbar section.

**Reply by Firegodd** [reply]
All but one bar. Can we get the option to turn them off in Ndui without turning off the micro menu, or a work around or something?

**Reply by siweia** [reply]
Download the version from github. You can disable actionbar in GUI with micromenu enabled.
